docs/commands/nop.md

```diff
@@ -18,36 +18,43 @@ instructions it overwrites
 `--b` Instead of replacing whole instructions, fill ITEMS bytes with nops
 
 nop the current instruction ($pc):
+
 ```bash
-gef➤ 	nop
+gef➤     nop
```

Comment:
These look weird now. Seems naively converting each tab to 4 spaces won't work on the `GEF_PROMPT` lines. We should probably just manually fix these to 2 spaces.

Reply:
We can fix in a later PR if you like.